server/src/db/bits/bits.schema.ts

```diff
@@ -30,6 +35,9 @@ const BitsSchema = new Schema<typeof methods>({
   downvotes: Number,
   title: String,
   content: String,
+  media: [{
+    uri: String,
```

Comment:
Can we also add some kind of source identifier? like `'YOUTUBE'` or `'TWITCH'`

Reply:
Is that not redundant with the URL? I see why it would be convenient, especially on the client, but it seems like we could easily derive it from the URL so I'm not sure we need to store it separately.